src/applications/disability-benefits/all-claims/tests/containers/Missing526Identifiers.unit.spec.jsx

```diff
@@ -98,4 +98,112 @@ describe('Form 526 Missing Identifiers Error Message', () => {
       );
     });
   });
+
+  describe('birlsId Missing', () => {
+    const props = {
+      title: 'File for disability compensation',
+      form526RequiredIdentifiers: {
+        participantId: true,
+        birlsId: false,
+        ssn: true,
+        birthDate: true,
+        edipi: true,
+      },
+    };
+
+    it('returns the missing birlsId OK message', () => {
+      const tree = render(<Missing526Identifiers {...props} />);
+      const messageParagraph = tree.container.querySelector('p');
+
+      expect(messageParagraph.textContent).match(
+        /It's ok if you don't know your BIRLS ID./,
```

Comment:
All instances of 'ok' should be uppercased to match casing in `okMessageToDisplayText`.